src/utils/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './util';
+export {ResizeWatcher} from './resize-watcher';
```

Comment:
ResizeWatcher is an internal utility, and is exposed via event to app, we don't plan to expose it for actual usage a utility.

Reply:
I need this one to be used in playkit-js-ui to support side panels correctly, I will explain in our meeting.

Reply:
OK, but I would advise you have a contingency plan as I don't see a reason to use more than one in the player core, and I see you already built upon it in UI manager.